src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -616,6 +640,36 @@ class PostgresEntityDataQueryService(
         }
     }
 
+    private fun deleteEntities(
+            entitySetId: UUID,
+            entities: Collection<UUID>,
+            partition: Int,
+            partitionVersion: Int
+    ): Int {
+        return hds.connection.use { connection ->
+            connection.autoCommit = false
```

Comment:
Shoudn't be autocommit set to true after committing?

Reply:
since the connection doesn't get closed I'm pretty sure you're right, but either way it seems safer to just reset it to true.

Reply:
We've found out that apparently when `.close()` is called on `connection` (by hikari in the case of `.use {`, the connection will actually be reset to its default state automatically (true)

Reply:
Good to know! Thanks!